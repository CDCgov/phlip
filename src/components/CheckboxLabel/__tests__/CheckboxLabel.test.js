import React from 'react'
import { shallow } from 'enzyme'
import { CheckboxLabel } from '../index'

const props = {
  input: {},
  label: 'Checkbox here',
  onChange: () => {},
  disabled: false,
  classes: {
    checked: {
      color: '#38a48e'
    }
  }
}

describe('CheckboxLabel component', () => {
  test('should render correctly', () => {
    expect(shallow(<CheckboxLabel {...props} />)).toMatchSnapshot()
  })
})