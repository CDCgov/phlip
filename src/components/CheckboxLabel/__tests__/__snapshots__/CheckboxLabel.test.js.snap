// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckboxLabel component should render correctly 1`] = `
<withStyles(FormControlLabel)
  control={
    <withStyles(Checkbox)
      checked={false}
      classes={
        Object {
          "checked": Object {
            "color": "#38a48e",
          },
        }
      }
      disabled={false}
      onChange={undefined}
    />
  }
  label="Checkbox here"
/>
`;
